R/meanimpute.R

```diff
@@ -1,4 +1,6 @@
 #' Meanimputation
+#' Calculates mean of a given vector, ignores NA values.
```

Comment:
This function actually **replaces** all NA values with the mean of a given vector.